### ISSUE TITLE
Radio focus remains on previously focused option (Firefox)

### ISSUE THREAD

**Issue by tokewf** [ngx-formly/ngx-formly#1631]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
For `radio` type fields, the focus remains on the previously focused option. This applies to Firefox 67.0.2 on macOS 10.14.5, but works fine in Chrome.

**Expected behavior**
Focus should correctly update to currently focused option.

**Minimal reproduction of the problem with instructions**
Open https://stackblitz.com/angular/xknldvdvpml or https://ngx-formly.github.io/ngx-formly/ui/material and scroll to "Radio type" _in Firefox_, and click the options.

* **Angular version:** 7.0.3

**Comment by aitboudad** [comment]
I couldn't reproduce it on Firefox `67.0.3` Ubuntu, I'll check on macOS later

**Comment by aitboudad** [comment]
fixed in `5.2.0`
